src/os/map/map.js

```diff
@@ -1,16 +1,19 @@
 goog.declareModuleId('os.map');
 
 import '../ol/ol.js';
+import FlightMode from '../map/flightmode.js';
```

Comment:
We should look into a lint rule to check for the shortest path to a module. From this module, you can use `./flightmode.js`.

Reply:
Ahh, heh yeah I just copied it in from the other file and didn't even think about it. I'll clean this up.